src/bindings/PyDP/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+include(FetchContent)
+
+FetchContent_Declare(
+  google_dp
+  GIT_REPOSITORY https://github.com/google/differential-privacy.git
+  GIT_TAG        main
```

Comment:
We should fix this to a commit, so that we know which one was used for creating the package